{"name": "Невозможность авторизоваться пользователем с некорректными данными", "status": "passed", "steps": [{"name": "Генерация данных для авторизации пользователя с некорректными полями", "status": "passed", "parameters": [{"name": "wrong_field", "value": "'password'"}, {"name": "login_data", "value": "{'email': 'francojoseph@example.net', 'password': '9p3t6ZdkK#'}"}], "start": 1712551277055, "stop": 1712551277083}, {"name": "Отправка запроса на авторизацию пользователя с некорректными данными", "status": "passed", "start": 1712551277083, "stop": 1712551277476}], "parameters": [{"name": "wrong_field", "value": "'password'"}], "start": 1712551277055, "stop": 1712551277477, "uuid": "5902054c-9cad-4f58-b3fb-fd8348cbeb5a", "historyId": "e1fcf4c3736e104004c2e0c2fe129470", "testCaseId": "cafda6e6a069b42edf40985093d528df", "fullName": "tests.tests_user.test_login_user.TestLoginUser#test_login_user_with_wrong_fields_failure", "labels": [{"name": "parentSuite", "value": "tests.tests_user"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-L83GKGS"}, {"name": "thread", "value": "9264-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_user.test_login_user"}]}